daemon/events/events.go

```diff
@@ -78,7 +78,7 @@ func (e *Events) Evict(l chan interface{}) {
 	e.pub.Evict(l)
 }
 
-// Log broadcasts event to listeners. Each listener has 100 millisecond for
+// Log broadcasts event to listeners. Each listener has 100 milliseconds for
```

Comment:
"Log a broadcast event to listeners. Each listener has 100 milliseconds to receive the event ..."